blocks/columns/columns.css

```diff
@@ -1,605 +1,675 @@
-    }
-
-    .section-top-grid > div:nth-of-type(2) > div:nth-of-type(2) {
-        margin-left: var(--spacer-element-05);
-        padding-top: var(--spacer-layout-05);
-    }
-
-    .columns.section-top-grid > div:nth-child(2) > div:nth-child(1) {
-        min-width: 620px
-    }
-
-    .section-top-grid > div > div:nth-child(odd) img {
-        width: 616px;
-        height: 616px;
-        max-width: 616px;
-    }
-
-    .section-top-grid > div > div:nth-child(even) img {
-        width: 262px;
-        height: 254px;
-        max-width: 262px;
-    }
-
-    .content-band.icon-list.picture-circle img {
-        border-radius: 50%;
-        height: 285px;
-        width: 285px;
-        }
-
-    .columns.content-band.icon-list.picture-circle > div > div {
-        margin-top: var(--spacer-layout-04);
-        }
-
-    .content-band.icon-list.picture-circle h3 {
-        margin: var(--spacer-layout-03) auto;
-        }
-
-    .section-top.columns-container {
-        background-size: 2600px 150%;
-        padding-bottom: var(--spacer-element-10);
-        padding-top: var(--spacer-layout-06);
-    }
-
-    .section-bottom.columns-container {
-        background-size: 2600px 239%;
-        padding-top: var(--spacer-layout-04);
-    }
-
-    .section.section-bottom .columns-wrapper .columns-2-cols {
-        padding-top: var(--spacer-layout-05);
-        padding-bottom: var(--spacer-layout-06);
-    }
-
-    /* teaser-list-manual min 1200 */
-    .teaser-list-manual .teaser-list {
-        position: relative;
-        display: grid;
-        grid-template-columns: repeat(2, 1fr);
-        gap: 0;
-    }
-
-    .teaser-list-manual .teaser-list .button-container {
-        margin-bottom: 25px;
-    }
-
-    .teaser-list-manual .teaser-list > div:nth-child(odd) > div > * {
-        margin-right: 50px;
-    }
-
-    .teaser-list-manual .teaser-list > div:nth-child(even) > div > * {
-        margin-left: 50px;
-    }
-
-    .teaser-list-manual .teaser-list div:last-child div:last-child {
-        border-bottom: 1px solid var(--neutral-sand);
-    }
-
-    .teaser-list-manual h4 {
-        font-size: var(--font-size-14);
-    }
-
-    .content-band.icon-list h4 {
-        font-size: var(--font-size-21);
-    }
-
-    .section.teaser-list-manual {
-        width: var(--section-width-desktop);
-        gap: 140px;
-        padding: var(--spacer-layout-07) 0;
-    }
-
-    .teaser-list-manual h6 {
-        font-size: var(--font-size-21);
-        line-height: 156%;
-    }
+  .section-bottom-grid > div > div {
+    flex: 1;
+    margin-left: var(--spacer-layout-05);
+  }
+
+  .section-top > div:first-child {
+    padding-top: 160px;
+  }
+
+  .columns.terms div {
+    margin: var(--spacer-element-06) 0;
+    column-gap: 120px;
+  }
+
+  .columns.quarter-circle > div {
+    gap: 120px;
+  }
+
+  .columns.quarter-circle img {
+    width: 552px;
+  }
+
+  .columns.stretch-last > div > div {
+    margin-left: 120px;
+  }
```

Comment:
This is putting a left margin on the 1st column (which isn't in the design).
![image](https://user-images.githubusercontent.com/3883795/231551407-bf849aa6-159a-4424-9295-6d448d5dad95.png)

Reply:
I'll switch to use `gap`.